src/CrossReferences.jl

```diff
@@ -214,8 +214,6 @@ end
 find_object(::Union{Function, DataType}, binding, ::Union{Union,Type{Union{}}}) = Utilities.Object(binding, Union{})
 find_object(other, binding, typesig) = Utilities.Object(binding, typesig)
 
-_method_exists(f, t) = method_exists(f, t)
```

Comment:
Apparently this actually made sense back when Documenter still supported 0.4. Good to get rid of this. :laughing: